### PR TITLE
308 random sample on stream

### PR DESCRIPTION
**Pull request by vlejd**
#308

### CODE REVIEW

File: gensim/test/test_wikicorpus.py [hunk 1]
```diff
@@ -62,6 +63,13 @@ def test_unicode_element(self):
         l = wc.get_texts()
         self.assertTrue(u'папа' in next(l))
 
+    def test_sample_text(self):
+        wc = WikiCorpus(datapath(FILENAME_U), processes=1)
+        print(len(wc))
```
Comment (menshikh-iv) [inline gensim/test/test_wikicorpus.py]: Why do we need this line?

File: gensim/test/test_wikicorpus.py [hunk 2]
```diff
@@ -62,6 +63,13 @@ def test_unicode_element(self):
...
+        for x in wc.sample_texts(1):
+            print(x)
```
Comment (menshikh-iv) [inline gensim/test/test_wikicorpus.py]: No needed to print result in tests. You can use `pass` instead.

File: gensim/test/test_wikicorpus.py (hunk 1 above)
Comment (vlejd) [inline gensim/test/test_wikicorpus.py]: This whole function should not be here. I removed it.

File: gensim/test/test_wikicorpus.py (hunk 2 above)
Comment (vlejd) [inline gensim/test/test_wikicorpus.py]: This whole function should not be here. I remove it.

**Comment by menshikh-iv** [comment]
Thank you @vlejd, сongratulations on your first PR :1st_place_medal:

**Pull request merged**